projects/observability/src/shared/components/topology/d3/d3-topology.ts

```diff
@@ -91,6 +95,21 @@ export class D3Topology implements Topology {
     this.hover = new TopologyHover(this.d3Util, this.domRenderer);
     this.click = new TopologyClick(this.d3Util, this.domRenderer);
     this.zoom = new TopologyZoom();
+    this.layout = this.initializeLayout(config.layoutType);
+  }
+
+  private initializeLayout(layoutType?: TopologyLayoutType): TopologyLayout {
+    switch (layoutType) {
+      case TopologyLayoutType.GraphLayout:
+        return new GraphLayout();
+      case TopologyLayoutType.TreeLayout:
+        return new TreeLayout();
+      case TopologyLayoutType.ForceLayout:
+        return new ForceLayout();
+      default:
+        return new CustomTreeLayout();
+    }
+    // TODO: Make this configurable with Node and edge renderers
```

Comment:
nit: is this comment relevant?

Reply:
I also don't know about that, since the parenthesis is still empty, I think we can level it there so that we don't forget about it, if we want to make that upgrade